package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "commit-and-tag-version",
-  "version": "12.0.0",
+  "version": "36.0.0",
```

Comment:
Will need to fix this, will have arisen from some tests clashing and trying to access the real files